04-vector-raster-integration-advanced.Rmd

```diff
@@ -292,13 +298,16 @@ plot(plot.locationSp_HARV,
 
 ```
 
-Looking at the plot we created in the challenge we see all the vegetation plot
-locations (blue) on the Canopy Height Model raster except for one.  Why? Refer
-back to our first figure in this lesson showing the relative extents of all the 
-spatial objects.  Notice that the plot extent (black) extends slightly farther 
-west than the CHM raster (green) extent.  Even though we assigned the extents 
-to match, there isn't any raster data that can appear in that area so it appears
-as white.  
+In the plot above, created in the challenge, all the vegetation plot locations
+(blue) appear on the Canopy Height Model raster layer except for one. One is
+situated on the white space. Why? 
+
+Look back at the first figure in this lesson showing the relative extents of
+all the spatial objects. Notice that the veg. plot extent (black) extends
+slightly farther west than the CHM raster (green) extent. We cropped the raster
+to the vector extent, but there isn't any raster data in that western region of
+our plot area so the vegetation plot in that area appears outside the extent of
+the raster data.
 
```

Comment:
I like the first paragraph (added it), but the second isn't true because there IS a plot outside the raster extent that is showing up in the plots.